iocuddle-sev/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::marker::PhantomData;
+
+use enumerate::enumerate;
+use iocuddle::*;
+use sev_types::command::{
+    GetIdentifier, PdhCertificateExport, PdhGenerate, PekCertificateImport,
+    PekCertificateSigningRequest, PekGenerate, PlatformReset, PlatformStatus,
+};
+
+// FIXME: https://github.com/rust-lang/rustfmt/issues/4085
+#[rustfmt::skip]
+enumerate! {
+    pub enum IoctlCode: u32 {
+	PlatformReset = 0,
+	PlatformStatus = 1,
+	PekGenerate = 2,
+	PekCertificateSigningRequest = 3,
+	PdhGenerate = 4,
+	PdhCertificateExport = 5,
+	PekCertificateImport = 6,
+	GetIdentifier = 7,
```

Comment:
> Can you repush so that cargo fmt succeeds and the eof newline test fails? Once I see that, I'll approve an additional patch to stop testing eof newlins on Rust code.

> Indentation here is wrong.

Letting `cargo fmt` do the wrong thing was the only way to get the CI to pass :sweat_smile:

Reply:
Why does `cargo fmt` work correctly [here](https://github.com/enarx/enarx/blob/master/intel-types/src/lib.rs#L73)?

Reply:
I don't know. I was curious about that too.